### PR TITLE
Minimize to tray support

### PR DESCRIPTION
**Pull request by djandreski**

### CODE REVIEW

File: app/main.js [hunk 1]
```diff
@@ -14,15 +18,42 @@ var mainWindowState = windowStateKeeper('main', {
 	height: 640
 });
 
+var appIcon = null;
 app.on('ready', function () {
+	showTrayIcon();
+	console.log("llllllllllll");
```
Comment (gamedevsam) [inline app/main.js]: Remove debug logging.

File: app/main.js [hunk 2]
```diff
@@ -1,10 +1,14 @@
 /* global __dirname */
+/* global requirejs */
```
Comment (gamedevsam) [inline app/main.js]: Is requirejs actually used or needed in this file?

**Comment by gamedevsam** [comment]
Should the tray icon go away when the window is re-opened?

**Comment by gamedevsam** [comment]
Also, this change is conflicting with a change made in another PR: https://github.com/Storj/driveshare-gui/pull/51

Please make sure to coordinate with other devs on who's supposed to do what on [Jixee](https://jixee.me/) to avoid duplication of efforts. Talk to Shawn if there is a workflow issue that needs to be addressed.

**Comment by super3** [comment]
@gordonwritescode This pull should be on the ```develop``` branch.

**Pull request closed without merging**